ONSdigital/eq-author
# Fixes broken staging deploy.

### What is the context of this PR?
A staging deploy was recently broken due to an update to a global dependency `serve`.
This change pins to a known working version of serve when building the Docker image.
The version used will now be controlled in the `package.json` file.

### How to review 
1. Checkout master
2. Build docker image with following command:
```
docker build -t onsdigital/eq-author .
```
3. Run the newly built docker image with the following command:
```
docker run -ti onsdigital/eq-author
```
4. Observe that an error is thrown.
5. Checkout this branchand repeat the above commands.
6. Observe there’s no error
